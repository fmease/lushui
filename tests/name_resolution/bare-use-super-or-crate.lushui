;; TEST fail check
;; the span information suck ... we should move the validator from the
;; name resolver to the lowerer which has more information (the pre-lowered form)

use super
use crate
use other

use crate.(self hello)
