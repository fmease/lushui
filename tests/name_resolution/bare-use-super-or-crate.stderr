error[E025]: `use` of bare `super` and `crate`
 >>> ${DIRECTORY}/name_resolution/bare-use-super-or-crate.lushui:5:1
  |
5 | use super
  | ^^^^^^^^^ 
  |
 help: add a name to it with `as`

error[E025]: `use` of bare `super` and `crate`
 >>> ${DIRECTORY}/name_resolution/bare-use-super-or-crate.lushui:6:1
  |
6 | use crate
  | ^^^^^^^^^ 
  |
 help: add a name to it with `as`

error[E025]: `use` of bare `super` and `crate`
 >>> ${DIRECTORY}/name_resolution/bare-use-super-or-crate.lushui:9:1
  |
9 | use crate.(self hello)
  | ^^^^^^^^^^^^^^^^^^^^^^ 
  |
 help: add a name to it with `as`

error: aborting due to 3 previous errors

